Notebooks/test new class.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
Yeah but I think this way it is currently easier to have the use-case notebook with the changes in the contacts class. This is thought of a convenience for me and testers. I will remove this before I merge the PR and collect the examples in the module docstring.